docs/Pipeline-Documentation.md

```diff
@@ -52,12 +60,36 @@ An assortment of other types of utility pipelines.  Utility piplines are named `
 
 ## Training
 
-Run model training on ground truth annotations you've created in order to generate a reusable pipe.
+Run model training on ground truth annotations.  Currently, training configurations are available to do object detection, object classification, and full-frame classification.  Tracker training will be added in a future update.
+
+### Important notes
+
+* Object detector training configurations require annotations labeled `fish`.
```

Comment:
If you train over generic detections I don't think you need the 'fish' label?

Reply:
Right, the default is to train on all labels encountered in groundtruth unless `--labels` are specified.

I forgot that `--input` is an older style and we have `-il, -it` now.  the `fish` I linked in slack was just an example placeholder.

The question remains though: Do _any_ of the training configurations require a "fish" label?  Which ones can take arbitrary ground truth, and which ones require specific labels?

Reply:
If you train over anything you shouldn't need the word fish no models would require it